packages/docs/playwright/gnosisSafeUtils.ts

```diff
@@ -25,22 +33,31 @@ export async function initGnosisSafe({ page, url }: { page: Page; url: string })
 
 export async function connectToMetamask({ page, context }: { page: Page; context: BrowserContext }) {
   const popupPromise = waitForPopup(context)
-  await page.click('//p[contains(text(), "Connect Wallet")]')
-  await page.click('//span[contains(text(), "Connect")]')
-  await page.click('//span[contains(text(), "MetaMask")]')
+  await page.click(XPath.text('p', 'Connect wallet'), { timeout: 2000 })
+  await page.click(XPath.text('button', 'Connect'))
+  await page.click('button:has-text("MetaMask")')
```

Comment:
Is there a reason why it's not consistent with others - `XPath.text('button', 'MetaMask')` ?